Xamarin.Forms.Xaml/TypeConversionExtensions.cs

```diff
@@ -153,9 +153,18 @@ public static object ConvertTo(this object value, Type toType, Type convertertyp
 			if (value != null)
 			{
 				var cast = value.GetType().GetRuntimeMethod("op_Implicit", new[] { value.GetType() });
-				if (cast != null && cast.ReturnType == toType)
-					value = cast.Invoke(null, new[] { value });
+				if (cast != null && cast.ReturnType == toType) {
+					value = cast.Invoke(null, new [] { value });
+					return value;
+				}
 			}
+
+			var nativeValueConverterService = DependencyService.Get<INativeValueConverterService>();
```

Comment:
this is the part where a `UIView` will be wrapped into a `UIViewWrapper`, fit to be included in a StackLayout